integration/test/performance_tracker/resources/config.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
One thing to consider going forward is if there is going to be collision when trying to get metrics from different performance tests that run concurrently. 

I ran into an issue when writing the CloudWatch Logs integration tests because I reused the log group and log stream name across all of the tests, so I had issues with getting consistent results from assertions because I was picking up data from multiple tests at the same time by accident. 

Not something that needs to be addressed right now, but should be something looked into in the next few weeks. The main focus is getting some performance test running and persisting data.